src/database/models/ApiKey.ts

```
@@ -0,0 +1,66 @@
+        where: eq(apiKeyTable.userId, userId),
+    });
+    return result;
+}
+
+export async function createApiKey(
+    userId: string,
+    keyHash: string,
+    name: string,
+    description: string,
+    keyId: string,
+): Promise<any> { // #TODO: Replace 'any' with the actual return type
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)